Controller/SystemInfoController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\Controller;
+
+use EzSystems\EzSupportToolsBundle\View\SystemInfoView;
+
+/**
+ * Do we need that controller at all ?
+ */
+class SystemInfoController
+{
+    public function viewInfoAction(SystemInfoView $view)
```

Comment:
Isn't this parameter a string - `systemInfoIdentifier` as defined in routing.yml? Or is some automagic taking care of it?